logbook-spring-boot-autoconfigure/src/main/java/org/zalando/logbook/autoconfigure/LogbookAutoConfiguration.java

```diff
@@ -323,6 +324,11 @@ public LogbookHttpResponseInterceptor logbookHttpResponseInterceptor() {
             return new LogbookHttpResponseInterceptor();
         }
 
+        @Bean
+        @ConditionalOnMissingBean(LogbookClientHttpRequestInterceptor.class)
```

Comment:
This should be mutually exclusive to the Apache HTTP Client setup (two bean methods above), shouldn't it?

Otherwise users will see requests and responses twice in the logs.